XAlloc.c

```diff
@@ -1,41 +1,48 @@
-
 #include "XAlloc.h"
-#include "RichString.h"
 
+#include "CRT.h"
+
+/* The function vasprintf is a GNU extension. */
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
```

Comment:
Isn't this set by `configure`/the `Makefile`?

Reply:
Indeed; replaced with a`#include "config.h"`